paddle/fluid/framework/framework.proto

```diff
@@ -166,6 +166,9 @@ message VarDesc {
   required string name = 1;
   required VarType type = 2;
   optional bool persistable = 3 [ default = false ];
+  // True if the variable is an input data and
+  // have to check the feed data shape and dtype
+  optional bool need_check_feed = 4 [ default = false ];
```

Comment:
We should discuss further whether to add this field inside `framework.proto`. It would be extremely hard to remove this field if we think that it is a bad design in the future.